src/AvatarStack.js

```diff
@@ -1,147 +1,149 @@
 import React from 'react'
+import classnames from 'classnames'
 import PropTypes from 'prop-types'
-import styled, {css} from 'styled-components'
+import styled from 'styled-components'
 import sx from './sx'
 import {get, COMMON} from './constants'
 import theme from './theme'
 
 const transformChildren = children => {
-  const count = children.length
   return React.Children.map(children, (child, index) => {
-    return (
-      <>
-        {count > 3 && index === 2 && <div className="AvatarItem-more AvatarItem" />}
-        {React.cloneElement(child, {className: 'AvatarItem'})}
-      </>
-    )
+    return <>{React.cloneElement(child, {className: 'pc-AvatarItem', sx: {zIndex: 10 - index}})}</>
```

Comment:
Two things:

1. If the child had any `className` or `sx` props set, this will overwrite them.
2. I don't think this has to be in a fragment, cause it's only one item:

```suggestion
    return React.cloneElement(child, {className: 'pc-AvatarItem', sx: {zIndex: 10 - index}})
```